vector/src/main/java/im/vector/app/features/onboarding/ftueauth/AbstractFtueAuthFragment.kt

```diff
@@ -86,21 +84,8 @@ abstract class AbstractFtueAuthFragment<VB : ViewBinding> : VectorBaseFragment<V
             is CancellationException                  ->
                 /* Ignore this error, user has cancelled the action */
                 Unit
-            is Failure.ServerError                    ->
```

Comment:
this has been lifted to the Login/Sign up fragment https://github.com/vector-im/element-android/pull/5517/files#diff-6422bfd992cde38d5783f859483c70b55ed045bc2c89a153784957ff2d0550e0R277 (the same change was made in `Login2`)